src/components/views/elements/PersistedElement.js

```diff
@@ -165,6 +165,7 @@ export default class PersistedElement extends React.Component {
 
         const parentRect = parent.getBoundingClientRect();
         Object.assign(child.style, {
+            zIndex: 10,
```

Comment:
I think we actually want this at 9 for when it can be dragged around (should show up under tooltips and things)
```suggestion
            zIndex: 9,
```